### ISSUE TITLE
Add support for Microsoft account creation and Bing.com searches for webscraping

### ISSUE THREAD

**Issue by John-Wick23** [sergerdn/py-bas-canvas-inspector-automator#4]
Add support for Microsoft account creation and Bing.com searches for webscraping as it is as much used as Google.com and hence a basic requirement.

**Comment by sergerdn** [comment]
Hi, @John-Wick23 

This project is designed exclusively for handling canvas requests during specific account creation processes. It is not intended for automating browsers or creating accounts, but solely for emulating aspects of these processes.

I have implemented it for Microsoft, please take a look:

https://github.com/sergerdn/py-bas-canvas-inspector-automator/blob/develop/py_bas_canvas_inspector_automator/automator/automator.py#L380

**Comment by John-Wick23** [comment]
> Hi, @John-Wick23
> 
> This project is designed exclusively for handling canvas requests during specific account creation processes. It is not intended for automating browsers or creating accounts, but solely for emulating aspects of these processes.
> 
> I have implemented it for Microsoft, please take a look:
> 
> https://github.com/sergerdn/py-bas-canvas-inspector-automator/blob/develop/py_bas_canvas_inspector_automator/automator/automator.py#L380

I did not mean implementing account creation, I meant implementation of canvas capturing during Microsoft Account creation and when searching on bing.com

**Comment by sergerdn** [comment]
> I did not mean implementing account creation, I meant implementation of canvas capturing during Microsoft Account creation 

This is done as I mentioned earlier.

> and when searching on bing.com

What was the reason for your request? Do Bing.com capture the canvas during the search?

**Comment by John-Wick23** [comment]
> Hi, @John-Wick23
> 
> This project is designed exclusively for handling canvas requests during specific account creation processes. It is not intended for automating browsers or creating accounts, but solely for emulating aspects of these processes.
> 
> I have implemented it for Microsoft, please take a look:
> 
> https://github.com/sergerdn/py-bas-canvas-inspector-automator/blob/develop/py_bas_canvas_inspector_automator/automator/automator.py#L380

Checked the code for Microsoft Account and it works perfectly as required. Can you add the feature to your pybas so that it auto captures the canvas fingerprint and updates it whenever any automation is done as it's a major issue for sites that have dynamic canvas fingerprinting implementation.

**Comment by John-Wick23** [comment]
> > I did not mean implementing account creation, I meant implementation of canvas capturing during Microsoft Account creation
> 
> This is done as I mentioned earlier.
> 
> > and when searching on bing.com
> 
> What was the reason for your request? Do Bing.com capture the canvas during the search?

Yes as far as I have observed the canvas data is captured on searching especially when you are signed into the account.

**Comment by sergerdn** [comment]
> Yes as far as I have observed the canvas data is captured on searching especially when you are signed into the account.

This project is not intended to create any account, only to emulate that process. 

But your request needs to create an account, logged in, and then search Bing, which is not the target of this project.
